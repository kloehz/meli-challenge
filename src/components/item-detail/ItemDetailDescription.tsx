interface IDesc {
    description: string
}

export const ItemDetailDescription = ({description}: IDesc) => {
    return (
        <div className="item-detail-description-container">
            <span className="desc-title">
                Descripcion del producto
            </span>
            <span className="desc-info">
                {description}
            </span>
        </div>
    )
}
