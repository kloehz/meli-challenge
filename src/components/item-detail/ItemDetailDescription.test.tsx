import { shallow } from "enzyme"
import { ItemDetailDescription } from "./ItemDetailDescription"

describe('ItemDetailDescription tests', () => {
    
    const wrapper = shallow(<ItemDetailDescription description={'descripcion'}/>);

    test('should be match snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });
    
    test('should be show the correct description', () => {
        expect(wrapper.find('.desc-info').text()).toBe('descripcion');
    })
    
})
