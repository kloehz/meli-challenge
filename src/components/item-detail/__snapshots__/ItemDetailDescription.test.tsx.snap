// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemDetailDescription tests should be match snapshot 1`] = `ShallowWrapper {}`;
